pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.bazaarvoice.commons</groupId>
         <artifactId>bv-opensource-super-pom</artifactId>
-        <version>1.0</version>
+        <version>1.5</version>
```

Comment:
Is this change relevant to the jackson upgrade? Or separate? If they are separate we should move it to separate pull requests.

Reply:
will revert that